charts/piraeus/crds/piraeus.linbit.com_linstorcontrollersets_crd.yaml

```diff
@@ -32,6 +32,10 @@ spec:
         spec:
           description: LinstorControllerSetSpec defines the desired state of LinstorControllerSet
           properties:
+            controllerEndpoint:
+              description: Cluster URL of the linstor controller. If not set, will
+                be determined from the current resource name.
+              type: string
```

Comment:
It's pretty redundant to have this in the controller set. However this way it is consistent across all 3 controllers. Users probably never need to set this value in any case.

Reply:
It is not great making our API messy for the sake of internal implementation details. Also, I don't think it makes that much difference to the code. We should just be able to take `ControllerEndpoint` out of `LinstorClientConfig` and put it directly in the CSI and node structs. Then we just pass it as an additional argument to `NewHighLevelLinstorClientFromConfig` and `NewClientConfigForAPIResource`.

It would trouble me less if it were documented as "internal, do not set" in the ControllerSet CRD, but I can't see how we can manage that without breaking the generation of CRDs from code.

Reply:
Done